src/app/shared/menu/menu-bar.component.css

```diff
@@ -0,0 +1,31 @@
+:host {
+  display: flex;
+  background-color: white;
+}
+/* TODO: not sure min-width should be dictated by prx-select, seems like this should be sized to the content */
```

Comment:
I'd make a ticket about this, but we aren't using prx-select in metrics at all once we get through the redesign, so it's not really a problem that we'll be solving for.